Sources/X509/CertificateSerialNumber.swift

```diff
@@ -84,3 +95,34 @@ extension Certificate.SerialNumber: CustomStringConvertible {
         return self.bytes.lazy.map { String($0, radix: 16) }.joined(separator: ":")
     }
 }
+
+@available(macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4, *)
```

Comment:
Note that this doesn't just want availability constraints, it needs a Swift version guard.

Reply:
Good catch, updated